src/UI/MainWindow.vala

```diff
@@ -155,6 +155,7 @@ namespace Spreadsheet.UI {
             scrolled.add (viewport);
 
             var sheet = new Sheet ();
+            sheet.set_size_request (2000, 2000);
```

Comment:
Maybe in `Sheet.draw` calculate the size depending on the number of columns and lines?